changelogs/fragments/285-postgresql_subscription_fix_idempontece.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - postgresql_subscription - Fix idempotence by casting the connparams dict variable
```

```suggestion
  - postgresql_subscription - fix idempotence by casting the ``connparams`` dict variable (https://github.com/ansible-collections/community.postgresql/issues/280).
```
To make it looking consistent with other entries